pages/about/translating.md

```diff
@@ -25,11 +25,13 @@ footer: |
 {% include box.html type="start" title="Summary" class="" %}
 {:/}
 
-This page provides general instructions on translating WAI resources.
+Help make Web Accessibility Initiative (WAI) resources available in your language.
 
-For a list of existing translations, see {% include link.html to="/translations/" text="All WAI Translations" %}.
+Join the volunteer translation community to raise awareness of digital accessibility among a global audience. This page provides general instructions on translating WAI standards and resources.
 
-To get announcements related to WAI translations, subscribe to the WAI Translations mailing list by sending e-mail to <a href="mailto:public-wai-translations-request@w3.org?subject=subscribe">public-wai-translations-request@w3.org with subject: &ldquo;subscribe&rdquo;</a>
+For a list of existing translations, see [All WAI Translations](/translations/).
+
+To get announcements related to WAI translations, subscribe to the WAI Translations mailing list by sending e-mail to [public-wai-translations-request@w3.org with subject: &ldquo;subscribe&rdquo;](mailto:public-wai-translations-request@w3.org?subject=subscribe)
```

Comment:
Perhaps this isn't worthy of being in the summary? I don't feel strongly either way. I just mentioned it in case you also thought so but just left it there from before?
Oh, perhaps even more importantly, we haven't been sending announcements of all new translations to that list, and I don't think we want to do that.

Reply:
> I just mentioned it in case you also thought so but just left it there from before?

Exactly.